test/set_test.c

```diff
@@ -248,6 +248,31 @@ void edit_test()
 
 void new_tag_test()
 {
+    const char* frame_id = "GEOB";
+    const char frame_data[8] = {0xD, 0xE, 0xA, 0xD, 0xB, 0xE, 0xE, 0xF};
```

Comment:
I didn't just use a string here because I thought it made it clearer that this isn't supposed to be testing text frames.